app-backend/batch/src/main/scala/com/azavea/rf/batch/export/spark/Export.scala

```diff
@@ -50,7 +50,7 @@ object Export extends SparkJob with Config with LazyLogging {
     projLocs: Map[UUID, List[(UUID, String)]],
     conf: HadoopConfiguration
   )(implicit sc: SparkContext): Unit = {
-    interpretRDD(ast, params.sources, ed.input.resolution, sceneLocs, projLocs) match {
+    interpretRDD(ast, params.sources, params.overrides, ed.input.resolution, sceneLocs, projLocs) match {
```

Comment:
@moradology where does metadata overriding occur? Should we be doing it the same way as this PR? If so, we can probably start passing `EvalParams` directly instead its fields individually.